pulsar-client-reactive-adapter/src/intTest/java/org/apache/pulsar/reactive/client/adapter/SingletonPulsarContainer.java

```diff
@@ -45,7 +45,7 @@ static PulsarClient createPulsarClient() throws PulsarClientException {
 	}
 
 	static DockerImageName getPulsarImage() {
-		return DockerImageName.parse("apachepulsar/pulsar:3.1.1");
+		return DockerImageName.parse("apachepulsar/pulsar:3.1.2");
```

Comment:
I am looking at ways to use the version in `lib.versions.toml` - but for now just keeping this manually in sync w/ the version.